src/main/kotlin/no/nav/familie/ks/sak/api/dto/VedtaksperiodeDto.kt

```diff
@@ -49,20 +51,27 @@ fun UtvidetVedtaksperiodeMedBegrunnelser.tilUtvidetVedtaksperiodeMedBegrunnelser
         fritekster = this.fritekster,
         utbetalingsperiodeDetaljer = this.utbetalingsperiodeDetaljer.map { it.tilUtbetalingsperiodeDetaljDto() },
         gyldigeBegrunnelser = this.gyldigeBegrunnelser.map { it.enumnavnTilString() },
+        eøsBegrunnelser = this.eøsBegrunnelser.map { it.tilEøsBegrunnelseDto(sanityBegrunnelser) }
     )
 }
 
-data class VedtaksbegrunnelseDto(
+data class BegrunnelseDto(
     val begrunnelse: String,
-    val vedtakBegrunnelseSpesifikasjon: String,
     val begrunnelseType: BegrunnelseType,
     val støtterFritekst: Boolean,
 )
 
-fun Vedtaksbegrunnelse.tilVedtaksbegrunnelseDto(sanityBegrunnelser: List<SanityBegrunnelse>) =
-    VedtaksbegrunnelseDto(
-        begrunnelse = this.nasjonalEllerFellesBegrunnelse.enumnavnTilString(),
-        begrunnelseType = this.nasjonalEllerFellesBegrunnelse.begrunnelseType,
-        støtterFritekst = this.nasjonalEllerFellesBegrunnelse.støtterFritekst(sanityBegrunnelser),
-        vedtakBegrunnelseSpesifikasjon = this.nasjonalEllerFellesBegrunnelse.enumnavnTilString(),
+fun NasjonalEllerFellesBegrunnelseDB.tilVedtaksbegrunnelseDto(sanityBegrunnelser: List<SanityBegrunnelse>) =
+    BegrunnelseDto(
+        begrunnelse = nasjonalEllerFellesBegrunnelse.enumnavnTilString(),
+        begrunnelseType = nasjonalEllerFellesBegrunnelse.begrunnelseType,
+        støtterFritekst = nasjonalEllerFellesBegrunnelse.støtterFritekst(sanityBegrunnelser),
```

Comment:
Det er mulig å jobbe på IBegrunnelse her:
```suggestion
fun IBegrunnelse.tilVedtaksbegrunnelseDto(sanityBegrunnelser: List<SanityBegrunnelse>) =
    BegrunnelseDto(
        begrunnelse = this.enumnavnTilString(),
        begrunnelseType = this.begrunnelseType,
        støtterFritekst = this.støtterFritekst(sanityBegrunnelser),
```

Også må vi endre linje 50 til `begrunnelser = this.begrunnelser.map { it.nasjonalEllerFellesBegrunnelse.tilVedtaksbegrunnelseDto(sanityBegrunnelser) },`
og linje 54 til `eøsBegrunnelser = this.eøsBegrunnelser.map { it.begrunnelse.tilEøsBegrunnelseDto(sanityBegrunnelser) }`

Reply:
Så slipper vi en funksjon